cli/util/convert.go

```diff
@@ -64,17 +66,25 @@ is set to the target transaction's NetworkFee value plus one (if it's sufficient
 conflicting transaction itself). If the target transaction is not in the memory pool, standard 
 NetworkFee calculations are performed based on the calculatenetworkfee RPC request. If the --gas 
 flag is included, the specified value is added to the resulting conflicting transaction network fee 
-in both scenarios.
+in both scenarios. When the --await flag is included, the command waits for one of the conflicting 
+or target transactions to be included in a block.
 `,
 					Action: cancelTx,
 					Flags:  txCancelFlags,
 				},
 				{
 					Name:      "txdump",
 					Usage:     "Dump transaction stored in file",
-					UsageText: "txdump [-r <endpoint>] <file.in>",
+					UsageText: "txdump [-r <endpoint>] <file.in> [--await]",
 					Action:    txDump,
 					Flags:     txDumpFlags,
+					Description: `Dumps the transaction from the given parameter context file to 
+   the output. This command expects a ContractParametersContext JSON file for input, it can't handle
+   binary (or hex- or base64-encoded) transactions. If transaction is signed and ready it can be dump 
+   to the given RPC node. If the -r flag is not included, the command only prints the transaction to
+   the console. If the --await flag is included, the command waits for the transaction to be included 
+   in a block before exiting.
```

Comment:
Replace starting from `If transaction is signed and ready...` to : If --rpc-endpoint flag is specified and transaction is signed then it will be sent to the network. If --await flag is specified, the command waits for the transaction to be included in a block before exiting.